index.js

```diff
@@ -25,34 +27,18 @@ app.get('/swagger.json', (req, res) => {
 });
 app.use('/docs', swaggerUI.serve, swaggerUI.setup(swagger.swaggerSpec));
 
-require('./models/User');
-
 app.use(router);
 
-// / catch 404 and forward to error handler
-app.use((_req, _res, next) => {
-  const err = new Error('Not Found');
-  err.status = 404;
-  next(err);
-});
-
-// no stacktraces leaked to user
-app.use((err, res) => {
-  if (err) {
-    return res.status(err.status || 500);
-  }
-
-  return res.json({
-    errors: {
-      message: err.message,
-      error: {}
-    }
+app.all('*', (_req, res) => {
+  res.status(400).json({
+    error: 'not found',
   });
 });
 
 // finally, let's start our server...
 const server = app.listen(process.env.PORT || 3000, () => {
-  console.log(`Listening on port ${server.address().port}`);
+  // eslint-disable-next-line no-console
+  console.log(`AH is Listening on port ${server.address().port}`);
```

Comment:
eslint rules shouldnot be toggled off in your files, do that in `.eslintrc`, and kindly dont commit files with `console.log()`